go.mod

```diff
@@ -0,0 +1,70 @@
+module github.com/lzap/packer-plugin-image-builder
+
+go 1.23.4
```

Comment:
something 1.22.x? probably not necessary to be in sync will the other osbuild repos, I guess. Ignore at will :-D

Reply:
As long as we are not shipping this in Fedora/RHEL this is fine I guess.